entries/m/michaels.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Michaels": {
+    "domain": "michaels.com",
+    "img": "michaels.com.png",
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "categories": [
+      "retail"
+    ]
```

```suggestion
    ],
    "regions": [
      "ca",
      "us"
    ]
```

Their [corporate about page](https://www.michaelspressroom.com/about) says they operate stores in the U.S. and Canada.

---

Good catch!